src/Networking/motor_interface.cpp

```diff
@@ -49,6 +67,76 @@ int getIndex(const std::vector<std::string> &arr, std::string &value)
   return -1;
 }
 
+bool isValidOperationMode(int motor_serial, int ik_axis, const std::string &op_mode)
+{
+  if (op_mode != "PWM target" && (motor_serial > 3 || motor_serial < 1) && ik_axis < 0)
```

Comment:
Might be better to use named constants in place of 3 and 1 here just so it's clearer what this does.